### ISSUE TITLE
can i run bat file using js?

### ISSUE THREAD

**Issue by cobola** [nwjs/nw.js#1069]
I have a bat file 

can i run the bat file?

<a href="javascript:runbat()">run</a>

function runbat(){
....
}

**Comment by vinimartinson** [comment]
Thank you so much Mithgol!

**Comment by SuperLukedin** [comment]
It works!! Thank you so much

**Comment by reshma119** [comment]
Its working, thanks!

**Comment by TheJaredWilcurt** [comment]
You should also wrap it in an OS check since `.bat` files only run on Windows
```js
if (process.platform === 'win32') {
  // your code
}
```

**Comment by tobysainsbury** [comment]
An important point to remember when calling other files from a .bat file with Node JS: You must ensure the directory of the command prompt window is correct.

I used Node JS to execute a .bat file, which executed another .bat file. This wasn't working (even though I specified full, absolute paths) until I realised that the first .bat file was being executed in Windows/System32.

I added a CD command to the start of the first .bat file, to change directory to the folder where the first .bat file was. The second file was then called successfully.
